Modules/Bridge/NumPy/wrapping/PyBuffer.i.init

```diff
@@ -17,6 +17,16 @@ try:
 
 except ImportError:
   HAVE_NUMPY = False
+try:
+    import numpy as np
+    from distributed.protocol import dask_deserialize
+    from typing import Dict, List
+    @dask_deserialize.register(NDArrayITKBase)
+    def deserialize(header: Dict, frames: List[bytes]) -> NDArrayITKBase:
+        loads = dask_deserialize.dispatch(np.ndarray)
+        return NDArrayITKBase(loads(header, frames))
+except ImportError:
+    pass
```

Comment:
It is typical to handle this as follows:

```suggestion
try:
    import numpy as np
    from distributed.protocol import dask_deserialize
    from typing import Dict, List
except ImportError:
    pass
else:
    @dask_deserialize.register(NDArrayITKBase)
    def deserialize(header: Dict, frames: List[bytes]) -> NDArrayITKBase:
        loads = dask_deserialize.dispatch(np.ndarray)
        return NDArrayITKBase(loads(header, frames))
```

Reply:
I'm not sure that registering a deserialization function would have much of an effect without also registering a serialization function.